File: 1-codes/micro/micro09.c
#import<stdio.h>

int main() {
  float preco, venda, novopreco;

  printf("Digite o preco: ");
  scanf("%f", &preco);

  printf("Digite o venda: ");
  scanf("%f", &venda);

  if(venda < 500 || preco < 30) {
      novopreco = preco + 10.0 / 100.0 * preco;
  } else if((venda >= 500 && venda <1200) || (preco >= 30 && preco <80)) {
      novopreco = preco + 15.0 / 100.0 * preco;
  } else {
    novopreco = preco + 20.0 / 100.0 * preco;
  }

  printf("O novo preco e %f", novopreco);
  return 0;
}

File: 6-gerador/tests/micro09.py
def main() -> None:
    preco = 0.0
    venda = 0.0
    novopreco = 0.0
    print("Digite o preco: ")
    float(input(preco))
    print("\nDigite a venda: ")
    float(input(venda))
    if venda < 500 or preco <30:
        novopreco = preco + 10/100 *preco
    elif (venda >= 500 and venda <1200) or (preco >= 30 and preco <80):
        novopreco = preco + 15/100 * preco
    elif venda >=1200 or preco >=80:
        novopreco = preco - 20/100 * preco

    print("\nO novo preco e: ")
    print(novopreco)

main()
